backend/Wheelingful.Core/Enums/UserRoleEnum.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Wheelingful.Core.Enums;
+
+public enum UserRoleEnum
+{
+    Admin,
```

Comment:
I would suggest reordering enums. In case UserRole is not set, it automatically becomes Admin.

Reply:
Good point.